presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -26,6 +26,7 @@
 import org.joda.time.DateTimeZone;
```

Comment:
Please change commit message to:
```
Allow to configure insert into Hive partition via configuration property
```
No need for anything more than that. Please do not mention internal references to issue tracking system.